cpu/stm32_common/stm32_mem_lengths.mk

```diff
@@ -233,7 +234,7 @@ else ifeq ($(STM32_TYPE), L)
     ifeq ($(STM32_MODEL2), 3)
       RAM_LEN = 64K
     else ifeq ($(STM32_MODEL2), 7)
-      RAM_LEN = 128K
+      RAM_LEN = 96K
```

Comment:
I think a comment or another variable for SRAM2 would be worth

Reply:
hmm wouldn't hurt but I tried to follow "don't add stuff that may be used later, but isn't now". I have no strong opinion on that though.